### DISCUSSION TITLE
Change map provider to Ordnance Survey

### DISCUSSION THREAD

**Discussion by OwenTheAnalyst** [wearefuturegov/outpost#269] [Ideas]
Public Sector Organisations get free access to consume API services from Ordnance Survey under the Public Sector Mapping Agreement (PSMA): https://beta.ordnancesurvey.co.uk/customers/public-sector/public-sector-licensing. Switching to Ordnance Survey as map provider may reduce costs from GoogleMaps and make it easier for organisations who themselves would not be able to use GoogleMaps (due to IG/procurement issues) to get set up using Outpost/Scout.

**Comment by JonathanDHolden** [comment]
Wouldn't this be an issue for users who want to then get directions to a service? I would think that the vast majority of people would want to use their phone, with Google Maps
